babybuddy/__init__.py

```diff
@@ -52,3 +52,7 @@
 VERSION = __version__
 
 default_app_config = 'babybuddy.apps.BabyBuddyConfig'
+
+import pymysql
+
+pymysql.install_as_MySQLdb()
```

Comment:
This would run for all configurations of the the app, regardless of the database used and that is not ideal. I _think_ it would be safe to move this `babybuddy/settings/mysql.py` but I'm not totally sure. I'm unclear on what the `install_as_MySQLdb` function does and if it would be OK to run multiple times?

Reply:
This worked! I'll commit it when I figure out the pipenv error above